website/docs/d/atracker_endpoints.html.markdown

```diff
@@ -0,0 +1,35 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_atracker_endpoints"
+description: |-
+  Get information about atracker_endpoints
+subcategory: "Activity Tracking API"
+---
+
+# ibm_atracker_endpoints
+
+Provides a read-only data source for atracker_endpoints. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax.
+
+## Example Usage
+
+```hcl
+data "ibm_atracker_endpoints" "atracker_endpoints" {
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+
+## Attribute Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - The unique identifier of the atracker_endpoints.
```

Comment:
chang the description of "id" i think its the time stamp not id .

Reply:
This is a uuid not a timestamp